### ISSUE TITLE
Fails when OpenMP isn't enabled

### ISSUE THREAD

**Issue by yurivict** [veg/hyphy#1503]
On systems where there is no OpenMP configure disables it and the build fails:
```
/wrkdirs/usr/ports/biology/hyphy/work/hyphy-2.5.40/src/core/tree.cpp:2760:9: error: use of undeclared identifier 'parallel'
    if (parallel.lLength) {
        ^
/wrkdirs/usr/ports/biology/hyphy/work/hyphy-2.5.40/src/core/tree.cpp:2770:28: error: use of undeclared identifier 'parallel'
        for  (id = 0; id < parallel.lLength; id++) {
                           ^
/wrkdirs/usr/ports/biology/hyphy/work/hyphy-2.5.40/src/core/tree.cpp:2771:29: error: use of undeclared identifier 'parallel'
            long matrixID = parallel.get (id);
                            ^
/wrkdirs/usr/ports/biology/hyphy/work/hyphy-2.5.40/src/core/tree.cpp:2780:24: error: use of undeclared identifier 'serial'
    for ( id = 0; id < serial.lLength; id++) {
                       ^
/wrkdirs/usr/ports/biology/hyphy/work/hyphy-2.5.40/src/core/tree.cpp:2781:25: error: use of undeclared identifier 'serial'
        long matrixID = serial.get (id);
                        ^
5 errors generated.
```

**Comment by spond** [comment]
Dear @yurivict,

Thanks for letting me know. I am working on a fix, and will include it in version 2.5.41 which I hope to release this week.

Best,
Sergei

**Comment by spond** [comment]
Dear @yurivict,

Should be fixed in `v2.5.41`

Best,
Sergei

**Comment by yurivict** [comment]
Dear Sergei,

Thanks for your prompt attention to this matter and for fixing the problem.
I submitted the request to update the FreeBSD port.

Best,
Yuri
